examples/tutorials/job_cli.ipynb

```diff
@@ -497,7 +498,7 @@
     "\n",
     "> the CLI argument\n",
     "> ```\n",
-    ">   -a dataset_path=\"/tmp/nvflare/data/cifar10\"\n",
+    ">   -a app_config=\"--dataset_path /tmp/nvflare/data/cifar10\"\n",
```

Comment:
It might be a bit confusing if "-a" without specify app name.

Should we just support "-f" and always use path to specify?